clash-lib/src/Clash/Backend/VHDL.hs

```diff
@@ -253,7 +253,8 @@ instance Backend VHDLState where
                 then emptyDoc
                 else line <> line <> renderAttrs (TextS.pack "signal") attrs) <> line <>
             nest 2
-              ("begin" <> line <>
+              (renderCompDecls cs
```

Comment:
I suppose I can piggyback my request for a change here since it is so intertwined with Alex's request.

My request was: Could you move `renderCompDecls` into the `nest` with the `decs` and `attrs`? The indentation is off. I mean, even more off than usual ;-).

However, Alex's requested changes would throw that all around and make the point moot.

Reply:
Oh I see! Thank you, I didn't notice that was possible.

Reply:
Btw I have checked and the identation works!